packages/common/src/hooks/useFeatureFlag.ts

```diff
@@ -74,9 +76,10 @@ export const createUseFeatureFlagHook =
     const setOverride = (value: OverrideSetting) => {
```

Comment:
Not super important but this should maybe be an async function now because setLocalStorageItem returns a promise

Reply:
yeah will update, good call